modules/endorspy/meta.yml

```diff
@@ -0,0 +1,54 @@
+name: "endorspy"
+description: endorS.py calculates endogenous DNA from samtools flagstat files and print to screen
+keywords:
+  - endogenous DNA
+  - ancient DNA
+tools:
+  - "endorspy":
+      description: "endorS.py calculates endogenous DNA from samtools flagstat files and print to screen"
+      homepage: "https://github.com/aidaanva/endorS.py"
+      documentation: "https://github.com/aidaanva/endorS.py"
+      tool_dev_url: "https://github.com/aidaanva/endorS.py"
+      doi: ""
```

Comment:
you could put the eager paper here if you wanted? But up to you if yo uwant to save it  for another publication if you want to publish it separtely